Assets/Scripts/Player/PlayerInventory.cs

```diff
@@ -189,6 +198,7 @@ void SpawnBuddy()
 
 		inventory[heldResources[resourceIndex]]--;
 		UpdateResourceList();
+		return true;
```

Comment:
I'd disagree only because that would put the entire body of the function within the if statement, whereas the way it is now keeps things a bit cleaner.